qobuz_importer.user.js

```diff
@@ -119,8 +129,9 @@ function insertLink(release) {
   var parameters = MBImport.buildFormParameters(release, edit_note);
 
   var mbUI = $('<p class="musicbrainz-import">' + MBImport.buildFormHTML(parameters) + MBImport.buildSearchButton(release) + '</p>').hide();
+  var isrcslist = $('<textarea>' + release.isrcs.join("\n") + '</textarea>');
```

Comment:
That's valuable information but not anyone may be interested in ISRCs, i'd rather provide a way to show/hide this textarea.
Something like http://jsfiddle.net/Ab9vX/3/ could do.
And this option should only be possible if the textarea isn't empty (i don't think all albums on Qobuz have ISRCs).